service.toml

```diff
@@ -44,6 +44,9 @@ optional = ["offset", "io_callback", "size", "encryption_customer_algorithm", "e
 [namespace.storage.op.write]
 optional = ["content_md5", "content_type", "io_callback", "storage_class", "encryption_customer_algorithm", "encryption_customer_key"]
 
+[namespace.storage.op.write_append]
+optional = ["content_md5", "content_type", "storage_class"]
```

Comment:
We can specify `content_type` and `storage_class` in `write_append`?

Reply:
Well, those values should be specified in the first write. But it's not a good idea to introduce these value in `write_append`. Can we set them in `create_append` like append 0 byte to object?

Reply:
If we can't do this, we need to save those value in the object and set them in the first time that user call `write_append`.

Something like the following:

```
offset, _ := o.GetAppendOffset()
if offset == 0 {
    value, _ = o.GetStorageClass()
}
```

Reply:
We can set them in `create_append` through append 0 byte to object.

Reply:
If we can't, we can set those metadata in `create_append` and use them while the user calls `write_append` the first time.